v2g-liberty/rootfs/root/v2g-liberty/appdaemon/apps/v2g-liberty/fm_client.py

```diff
@@ -449,14 +501,16 @@ async def trigger_schedule(self, *args, **fnc_kwargs) -> str:
                 "roundtrip-efficiency": c.ROUNDTRIP_EFFICIENCY_FACTOR,
                 "power-capacity": str(c.CHARGER_MAX_CHARGE_POWER) + "W"
             },
-            "flex-context": self.FM_OPTIMISATION_CONTEXT,
+            "flex-context": c.FM_OPTIMISATION_CONTEXT,
         }
-
-        res = requests.post(
-            url,
-            json=message,
-            headers={"Authorization": self.fm_token},
-        )
+        try:
+            res = requests.post(
+                url,
+                json=message,
+                headers={"Authorization": self.fm_token},
+            )
+        except:
+            self.log(f"Trigger_schedule on url '{url}' failed.")
```

Comment:
Perhaps also log the error here.

.. Or, I guess you'll soon use the Client here anyway.